tv-shows-app/src/components/features/review/RatingStars.tsx

```diff
@@ -0,0 +1,40 @@
+import { StarIcon } from "@chakra-ui/icons";
+import { Flex, Text } from "@chakra-ui/react";
+
+interface IStarRatingProps {
+  label: string;
+  onChange: (clicked: boolean, index: number) => void;
+  value: {
+    selected: number;
+    hovered: number;
+  };
+}
+
+export default function ({ label, onChange, value }: IStarRatingProps) {
+  return (
+    <>
+      <Flex flexDirection="row" gap={2}>
+        <Text color="white" fontSize={20} marginRight={2}>
+          {label}
+        </Text>
+        {[...Array(5)].map((_, index) => {
```

Comment:
I'd prefer to use real variable names instead of `_`. Did you see this somewhere? :)